### ISSUE TITLE
[5.0.0alpha3] Bartlett\Reflect\Environment class not found

### ISSUE THREAD

**Issue by remicollet** [llaville/php-reflect#33]
but is used in some tests

```
1) Bartlett\Tests\Reflect\EnvironmentTest::testUndefinedScanDir
Error: Class 'Bartlett\Reflect\Environment' not found

```

**Comment by llaville** [comment]
I know, unit tests should be refactored as documentation (before reaching beta or RC stage : next steps on my roadmap)

**Comment by llaville** [comment]
I'll fix the remaining failed tests (Environment) tomorrow. All others are now running with commit ed617848a3fcf5d3940dd2f35ca28a04e985e20b

**Comment by llaville** [comment]
Fixed now with release 5.0.0-beta1
